iterative/utils/helpers.go

```diff
@@ -23,3 +23,13 @@ func SetId(d *schema.ResourceData) {
 		}
 	}
 }
+
+func StripAvailabilityZone(region string) string {
+	lastChar := region[len(region)-1]
+	// 0x61(a) to 0x7a(z)
+	if lastChar >= 0x61 && lastChar <= 0x71 {
```

Comment:
```suggestion
	if lastChar >= 'a' && lastChar <= 'z' {
```

Using [`rune` literals](https://go.dev/ref/spec#Rune_literals) could be a bit clearer and reduce the need for explanatory comments.